tests/karma.conf.js

```diff
@@ -1,11 +1,21 @@
+require('dotenv').load({silent: true});
```

Comment:
This is required to let developers add their Browserstack user and key to the `.env` file.